include/ripples/configuration.h

```diff
@@ -74,7 +75,9 @@ struct GraphInputConfiguration {
         ->group("Input Options")
         ->required();
     app.add_flag("--metall-store-dir", metall_dir,
-                 "Directory to store metall graph data.")
+                    "Directory to store metall graph data.")
+        ->group("Input Options");
+    app.add_flag("--rr-store-dir", rr_dir, "Directory to store RR data.")
```

Comment:
Flag option for RR set storage - compatible with both memkind and metall implementations.